### ISSUE TITLE
Update request: rtx 1.34.1 → 1.35.1

### ISSUE THREAD

**Issue by x80486** [NixOS/nixpkgs#245979]
- Package name: `rtx`
- Latest released version: `1.35.1`

- Current version on the unstable channel: `1.34.1`
- Current version on the stable/release channel: `1.30.1`

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

## Notify Maintainers

@konradmalik

-----

Note for maintainers: Please tag this issue in your PR.

**Comment by x80486** [comment]
I actually tried to send a pull request, but I couldn't figure it out what's the correct hash, and for what file:

```nix
src = fetchFromGitHub {
  owner = "jdxcode";
  repo = "rtx";
  rev = "v${version}";
  sha256 = "???";
};
```

**Comment by babeuh** [comment]
If you leave it blank and try to build it, it will give you the hash.

**Comment by antonmosich** [comment]
This is already closed by #247007
